bundles/org.openhab.binding.somfymylink/src/main/java/org/openhab/binding/somfymylink/internal/model/SomfyMyLinkCommandShadeParams.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.somfymylink.internal.model;
+
+/**
+ * @author Chris Johnson - Initial contribution
+ */
+public class SomfyMyLinkCommandShadeParams extends SomfyMyLinkCommandParamsBase {
+    
+    public SomfyMyLinkCommandShadeParams(String targetId, String auth) {
+        super(auth);
+        this.targetId = targetId.replace('-', '.');
+    }
+
+    String targetId = "";
```

Comment:
This is populated by the constructor anyway.
```suggestion
    String targetId;
```

Reply:
done. and made public final + moved in file to correct location.